### DISCUSSION TITLE
How to use SLR (sparse logistic regression) classification with openNFT?

### DISCUSSION THREAD

**Discussion by brainwatcher** [OpenNFT/OpenNFT#124] [Q&A]
I am new in NF-FMRI. I would like to apply the result of SLR of ROI in my NF experiment. I post this question to ask the steps of using SLR. What should I do after I made the classifier. Should I translate the SLR classifier into a SVM weight matrix? The chosen voxel with non-zeros weight while the other zeros. Is this the usual way?

**Comment by ykoush** [comment]
SLR sounds as a stand-alone method. You can use OpenNFT and its SVM feedback type routines (isSVM) to implement/modify towards your own feedback type.

**Reply by brainwatcher** [reply]
Thank you for you reply in discussion. SLR is sparse logistic regression. I read it from _"Visual image reconstruction from human brain activity using a combination of multiscale local image decoders"_. It is a classifier like SVM but got sparse voxel selection. Some article use it to decode ROI in fMRI-NF.  I would like to try this model with openNFT.  And it has some algorithm improvements like grouped SLR. (_"Grouped sparse Bayesian learning for voxel selection in multivoxel pattern analysis of fMRI data"_.) I think this maybe an important development direction of fMRI-NF. 
As for the implement, I think it has a linkage function as logistic regression.  So I need try to learn how to modify the SVM routine in openNFT.
